### ISSUE TITLE
Toggle Passing Keys in NVDA

### ISSUE THREAD

**Issue by huanlin** [nvaccess/nvda#12022]
### Is your feature request related to a problem? Please describe.

I simply copy and paste the requirement from a post at a [NVDA discussion group](https://nvda.groups.io/g/nvda/topic/toggle_passing_keys_in_nvda/74088934?p=) because I need the same thing, as below:

> I was wondering if anyone knew of a way to tell NVDA to stop processing commands for a time, similar to the "pass next key through" command. When using an application/ widget that has heavy keyboard accessibility, NVDA's commands really hinder the use of the application, and having to press NVDA+F2 between each and every keystroke would make - say - a game using its own keyboard commands practically unplayable. I thought perhaps the "application sleep mode" was what I needed, but not only does the command listed on the reference do absolutely nothing for me, it apparently stops the speech synthesis as well, which I don't want.
>
> What I'm looking for is a key stroke that tells NVDA to start ignoring/passing keys until that same keystroke is entered again, or something to that effect.
>
>Searching for this has yielded nothing helpful because any way I phrase the question, I simply get linked back to the user guide or commands quick reference, which so far as I've seen, doesn't have any solution. Is there perhaps an addon that would allow this, or have I missed something in NVDA's main feature set?

### Describe the solution you'd like

To provide toggle mode for passing keys in NVDA.

### Describe alternatives you've considered

Sleeping mode did not solve my problem. However, an addon named [Numpad Nav Mode](https://github.com/opensourcesys/numpadNavMode) fit my needs. It would be great if NVDA provides this feature.

### Additional context

Source discussion post: https://nvda.groups.io/g/nvda/topic/toggle_passing_keys_in_nvda/74088934?p=

**Comment by huanlin** [comment]
After created this ticket, I found another one that is created at 2017: #7062

**Comment by leonardder** [comment]
Closing this one in favour of #7062
